ui/app/src/state/epics/auth.ts

```diff
@@ -58,7 +58,8 @@ const epics: CrafterCMSEpic[] = [
         return [
           messageSharedWorker(refreshAuthToken()),
           // This action here assumes this epic is ran only when login from session timeout, not on initial login too
-          hasActiveSite && messageSharedWorker(openSiteSocket({ site: state.sites.active, xsrfToken: getXSRFToken() }))
+          hasActiveSite && messageSharedWorker(openSiteSocket({ site: state.sites.active, xsrfToken: getXSRFToken() })),
+          messageSharedWorker(openSiteSocket({ xsrfToken: getXSRFToken() }))
```

Comment:
Find opportunities on how to auto connect to global topic without the UI explicit or use devoted action